src/CicadaVote.sol

```diff
@@ -321,7 +323,7 @@ abstract contract CicadaVote {
         internal
         view
     {
-        bytes32 parametersHash = keccak256(abi.encode(pp));
+        bytes32 parametersHash = keccak256(abi.encode(pp, msg.sender));
```

Comment:
I think we can leave this one as is, since this function is only used to finalize a vote.

Reply:
Yeah, I agree. Fixed. Everyone should be able to finalize to vote, once the voting period has ended.